### ISSUE TITLE
Tag aliases for deratives of root word

### ISSUE THREAD

**Issue by JSG-1** [pjeby/tag-wrangler#127]
I've been using your Tag Wrangler plugin for Obsidian and I must say, it's fantastic. It has significantly enhanced my note-taking process, and I appreciate the effort you've put into developing it.

However, I've come across a challenge that I couldn't solve with the available documentation on GitHub. It's about tag management, specifically regarding aliasing tags. In my workflow, I often find myself using the tag #accountable interchangeably with #accountability. This is purely for the sake of writing fluidity, but it leads to some inefficiency in managing and searching tags due to the need to remember and search for both variations.

I was wondering if Tag Wrangler provides a way to alias one tag to another, effectively grouping them under a single tag for organizational purposes. This feature would streamline my workflow and, I believe, add valuable functionality to your already impressive plugin.

Thank you again for your dedication to the Obsidian community through your work on Tag Wrangler. I look forward to your guidance on this matter.

**Comment by pjeby** [comment]
Yes - sort of, but only in the sense that you can replace tags with tag pages -- since pages (notes) can have aliases.

So let's say you make a note with aliases `accountable` and `accountability`.  You can then just type `[[account` and get autocompletion for a link to that note with the correct alias, which will appear with the correct ending in context.

Tag Wrangler lets you convert tags to notes in this fashion, so you could actually create tag pages for both tags, or a single note with aliases for `#accountable` and `#accountability`, and then use Obsidian's "unlinked references" feature to search for the references and change them to links to the relevant notes, finally merging the two notes to consolidate the links.

(But of course, you could also do it manually.)

Anyway, tags can't actually be aliased in Obsidian, but notes can.  If the reason you're using tags right now is the ease of creating new ones, or the presence of the tags pane, there are some workarounds for that.  Obsidian does allow you to link to non-existent pages and create the content later.  Also, you can use dataviews, incoming link views, and embeddable searches to create your own custom organizational views to replace most of the functionality of the tags pane, and perhaps even improve on it, depending on what precisely you're looking to do.
